.github/workflows/pr-comment-bot.yml

```
@@ -43,6 +43,7 @@ jobs:
 
             * 🙏 The Clarity team thanks you for opening a pull request
             * ⏳ The build for this PR has started
+            * 🍿 In the meantime, explore it in [stackblitz](https://pr.new/github.com/${{steps.get-pr-event.outputs.sourceHeadRepo}}/pull/${{steps.get-pr-event.outputs.pullRequestNumber}})
```

```suggestion
            * 🍿 In the meantime, explore it in [StackBlitz](https://pr.new/github.com/${{steps.get-pr-event.outputs.sourceHeadRepo}}/pull/${{steps.get-pr-event.outputs.pullRequestNumber}})
```

StackBlitz should proper casing.

---

corrected in commit 502e5d4